<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frameBg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>282</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameBg {
	
	color: rgb(100, 100, 100);
	background-color: rgb(245, 245, 245);

	border: 1px solid #c7cacd;
        border-radius: 4px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../one.qrc">:/3/res/界面.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameLoginRegister">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>311</width>
      <height>282</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameLoginRegister {
	
	color: rgb(100, 100, 100);
	background-color: rgb(245, 245, 245);

	border: 1px solid #c7cacd;
        border-radius: 4px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonRegisterPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
       <property name="icon">
        <iconset resource="../one.qrc">
         <normaloff>:/3/res/编辑.png</normaloff>:/3/res/编辑.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonLoginPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
       <property name="icon">
        <iconset resource="../one.qrc">
         <normaloff>:/3/res/填选购人.png</normaloff>:/3/res/填选购人.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>用户密码：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButtonForget">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>忘记密码</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/提示.png</normaloff>:/3/res/提示.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QPushButton" name="pushButtonLogin">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/填选购人.png</normaloff>:/3/res/填选购人.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditUsername">
           <property name="placeholderText">
            <string>邮箱地址/账号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>账    号：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>记住</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>账    号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditRegisterName">
           <property name="placeholderText">
            <string>用户账号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>邮    箱：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditEmail">
           <property name="placeholderText">
            <string>邮箱地址</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>用户密码：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditRegisterPassword1">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>确认密码：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditRegisterPassword2">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>确认密码</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>验 证 码：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditVerify">
           <property name="placeholderText">
            <string>邮箱验证码</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="pushButtonSend">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/confirm.png</normaloff>:/3/res/confirm.png</iconset>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>注册后直接登录</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QPushButton" name="pushButtonRegister">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/编辑.png</normaloff>:/3/res/编辑.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>账    号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditEmail2">
           <property name="placeholderText">
            <string>用户账号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>新 密 码：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditNewPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>确认密码：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditNewPassword2">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>确认密码</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>验 证 码：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="placeholderText">
            <string>邮箱验证码</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButtonVerify2">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/confirm.png</normaloff>:/3/res/confirm.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLabel" name="label_14">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>找回密码后直接登录</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QPushButton" name="pushButtonForgetOk">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid #c7cacd;
        border-radius: 4px;
    }
    QPushButton:pressed {
        border: 1px solid #c7cacd;
    }</string>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
           <property name="icon">
            <iconset resource="../one.qrc">
             <normaloff>:/3/res/confirm.png</normaloff>:/3/res/confirm.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{
background-color:rgb(76, 114, 176)
}
QPushButton{
border:none;
	background-color:rgba(0,0,0,0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);
font: 9pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>AI4TSF</string>
       </property>
       <property name="icon">
        <iconset resource="../one.qrc">
         <normaloff>:/3/res/数据质控.png</normaloff>:/3/res/数据质控.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/3/res/minus.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/3/res/closev.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../one.qrc"/>
 </resources>
 <connections/>
</ui>
